### ISSUE TITLE
Establish Python Developer skill level

### ISSUE THREAD

**Issue by relativityboy** [steemit/devportal#143]

**Comment by feruzm** [comment]
**Results**

1. Python developer should be familiar with following versions of Python 2.7, Python 3.3 and 3.4,  3.5, 3.6.

2. Also medium knowledge of package manager `pip` and/or `pipenv` is required, since our tutorials will be using package manager to get started quickly.

3.  Terminal (Linux/Mac) or Cmd (Windows) knowledge is also required, since most python tutorials will only work with terminal and print out result in different formats. Python environment path should be set up properly so that terminal can access proper python library without conflicts.

4. Additional packages (optional), setuptools, brew

**Comment by relativityboy** [comment]
@feruzm - Python 2 has been a drag on the Py community for years... maybe for simplicity and clarity, and a better developer experience we should encourage developers to use python 3 and focus our tutorials there. 

What are your thoughts? Are there strong deal-breakers re-not supporting Python 2?

**Comment by feruzm** [comment]
@relativityboy I agree, it is good to have support for python 2 but I also thought tutorials will be in python 3.

**Comment by relativityboy** [comment]
@feruzm - Cool. Can you update the results comment to reflect that?
